tasks/enableL1TokenAcrossEcosystem.ts

```diff
@@ -9,13 +9,13 @@ const NO_SYMBOL = "----";
 const NO_ADDRESS = "------------------------------------------";
 
 // Supported mainnet chain IDs.
-const enabledChainIds = Object.values(MAINNET_CHAIN_IDs)
-  .map(Number)
-  .filter((chainId) => chainId !== CHAIN_IDs.BOBA)
-  .sort((x, y) => x - y);
-
-const chainPadding = enabledChainIds[enabledChainIds.length - 1].toString().length;
-const formatChainId = (chainId: number): string => chainId.toString().padStart(chainPadding, " ");
+const enabledChainIds = (hubChainId: number) => {
```

Comment:
We should probably read this from a config store to be safe, just a comment

Reply:
There's a subtle bootstrapping problem with that - it doesn't work for new chains because the entries don't exist at the point where we're using this script to generate the routes :(